src/CMakeLists.txt

```diff
@@ -46,8 +46,11 @@ set(BA_SOURCES
     ${CMAKE_CURRENT_SOURCE_DIR}/base_alloc/base_alloc_linear.c
     ${CMAKE_CURRENT_SOURCE_DIR}/base_alloc/base_alloc_global.c)
 
+set(CTL_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/ctl/alloc.c
+                ${CMAKE_CURRENT_SOURCE_DIR}/ctl/ctl.c)
+
 if(LINUX)
-    set(BA_SOURCES ${BA_SOURCES}
+    set(BA_SOURCES ${BA_SOURCES} ${CTL_SOURCES}
```

Comment:
should CTL sources be a part of BA sources var?

Reply:
Done